vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/validation/MultiSelectComboBoxBinderValidationIT.java

```diff
@@ -0,0 +1,88 @@
+package com.vaadin.flow.component.combobox.test.validation;
+
+import com.vaadin.flow.component.combobox.testbench.MultiSelectComboBoxElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.validation.AbstractValidationIT;
+import org.junit.Test;
+import org.openqa.selenium.Keys;
+
+import static com.vaadin.flow.component.combobox.test.validation.ComboBoxBinderValidationPage.*;
```

Comment:
```suggestion
import static com.vaadin.flow.component.combobox.test.validation.MultiSelectComboBoxBinderValidationPage.*;
```

Reply:
Let's use explicit imports instead of wild card ones.

Reply:
Done